_scripts/instruction-widget/templates/install/pip.html

```diff
@@ -23,21 +23,43 @@ <h3>Partial support</h3>
       the cryptography project's site</a> for more infoformation.</p>
 
     <p>Commands to install system dependencies may look like the following, run on the command
-    line on the machine:
-      <pre class="no-before"><ol><li># For APT-based distributions (e.g. Debian, Ubuntu ...)</li>
+    line on the machine.</p>
+    <p>For APT-based distributions (e.g. Debian, Ubuntu ...):
+      <pre class="no-before"><ol>
         <li>sudo apt update</li>
-        <li>sudo apt install python3-dev python3-venv libaugeas0</li>
-        <li># For RPM-based distributions (e.g. Fedora, CentOS ...)</li>
-        <li># NB1: old distributions will use yum instead of dnf</li>
-        <li># NB2: RHEL-based distributions use python3X-devel instead of python3-devel (e.g. python36-devel)</li>
-        <li>sudo dnf install python3-devel augeas-libs redhat-rpm-config</li></ol></pre>
+        <li>sudo apt install python3 python3-venv libaugeas0</li>
+      </ol></pre>
+    </p>
+    <p>For RPM-based distributions (e.g. Fedora, CentOS ...):
+    <br>NB1: old distributions will use yum instead of dnf
+    <br>NB2: RHEL-based distributions use <code>python3X</code> instead of <code>python3</code>
```

Comment:
nit: I personally think this "NB#: ..." text from https://certbot.eff.org/docs/contributing.html#running-a-local-copy-of-the-client looks kind of sloppy here. If you agree, can we reformat this maybe as a note with more or less complete sentences? If you disagree, feel free to leave it as is.

Reply:
I agree, will give it a try.